Formula/libpython-tabulate.rb

```diff
@@ -1,8 +1,8 @@
 class LibpythonTabulate < Formula
```

Comment:
https://github.com/Homebrew/homebrew-core/pull/112518#issuecomment-1272512678

Reply:
Updated, still need to figure out which runtime python to use that doesn't cause audit failures. The following fails with `References to python@3.9 should match the specified python dependency`:

```ruby
depends_on "python@3.9" => [:build, :test]
depends_on "python@3.10"
```

With `uses_from_macos`, I'm wondering how to refer to the runtime python in the install block (I don't think we can simply use PATH's python)

```ruby
depends_on "python@3.9" => [:build, :test]
depends_on "python@3.10" => [:build, :test]

uses_from_macos "python"
```

Any ideas?